src/components/CallbackForm/CallbackForm.js

```diff
@@ -168,9 +174,23 @@ export default function CallbackForm({
         }
         if (callTypes.includes(value)) {
             setHelpNeeded(value);
+            setSubtypeDropdownValues(value);
```

Comment:
This is an existing issue but the function this is in feels like it should be 2 functions as it's now got 2 reasons to change.